src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -502,14 +507,22 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
 
   private getHTMLContent(): string {
     if (this.cells.length === 1 && this.cells[0].length === 1) {
+      if (!this.cells[0][0]) {
+        return "";
+      }
       return this.getters.getCellText(this.cells[0][0].position);
     }
 
     let htmlTable = '<table border="1" style="border-collapse:collapse">';
     for (const row of this.cells) {
       htmlTable += "<tr>";
       for (const cell of row) {
-        const cssStyle = cssPropertiesToCss(cellStyleToCss(cell.style));
+        if (!cell) {
+          continue;
+        }
+        const cssStyle = cssPropertiesToCss(
+          cellStyleToCss(this.getters.getCellComputedStyle(cell.position))
```

Comment:
Isn't this another fix (that should be in another commit, and possibly backported to earlier versions) ? That because we don't copy the conditional formats, right ?

Reply:
No, in fact this change is coming from the fact that, now, cell.style is not present anymore while, before, we were pushing `getCellComputedStyle(cell.position)` in the object when copying the cells. So it's not a fix, only a change caused by my other changes :)